### PR TITLE
Finally fix stack-chain issue (closes #2166)

### LINKED ISSUE #2166: Promise rejection because of stack-chain when different global and local TestCafe versions are installed
### Are you requesting a feature or reporting a bug?
Bug

### What is the current behavior?
Crash when older (`0.18.6`) TestCafe is installed globally:
```
Using locally installed version of TestCafe.
(node:18156) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): Error: Conflicting version of stack-chain found
(node:18156) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

### What is the expected behavior?
No crash

### How would you reproduce the current behavior (if this is a bug)?
Install `0.18.6` globally and `0.19.0-alpha1` locally, try to run any test.

### Specify your

* operating system: Windows 10
* testcafe version: 0.19.0-alpha1 && 0.18.6
* node.js version: 8.9.4

### PR DESCRIPTION
**Pull request by AndreyBelym**
\cc @miherlosev

### CODE REVIEW

File: src/cli/index.js [hunk 1]
```diff
@@ -1,189 +1,70 @@
...
+    terminationHandler.on(TerminationHandler.TERMINATION_LEVEL_INCREASED_EVENT, () => childCli.send('shutdown'));
 
-    try {
-        var argParser = new CliArgumentParser();
+    childCli.on('exit', code => process.exit(code));
+}
 
-        await argParser.parse(process.argv);
+(async function loader () {
+    const needNewProcess = isNewProcessNeeded();
```
Comment (miherlosev) [inline src/cli/index.js]: More clear
```js
const inNewProcess = shouldRunInNewProcess();
....
return startCli(cliPath, inNewProcess);
```

File: src/cli/index.js [hunk 2]
```diff
@@ -1,189 +1,70 @@
...
-import createTestCafe from '../';
+import TerminationHandler from './termination-handler';
 
 
-const TERMINATION_TYPES = {
-    sigint:   'sigint',
-    sigbreak: 'sigbreak',
-    shutdown: 'shutdown'
-};
+const SAFE_VERSION = '0.19.0-alpha2';
```
Comment (miherlosev) [inline src/cli/index.js]: `SAFE_VERSION` is not clear name.

Something like
`const MINIMAL_VERSION_WITH_SINGLE_STACK_CHAIN_MODULE_IN_PROCESS =...`
Also, add comment why we do this.

**Comment by AndreyBelym** [comment]
FPR \cc @miherlosev @helen-dikareva

File: package.json [hunk 3]
```diff
@@ -161,6 +160,7 @@
     "run-sequence": "^1.2.2",
     "saucelabs-connector": "^0.2.0",
     "serve-static": "^1.10.0",
+    "stack-chain": "^2.0.0",
```
Comment (miherlosev) [inline package.json]: Is this dependency necessary?

Comment (AndreyBelym) [reply package.json]: It's left in [tests](https://github.com/AndreyBelym/testcafe/blob/6e70243a736f174c36cd89d6f2abac2de73e3ee8/test/server/data/test-callsite/index.js) to compare callsites produced with `stack-chain` and without it.

**Pull request merged**